itemv2.js

```diff
@@ -100,7 +120,12 @@ const BaseItem = {
 
 const BillItem = {
   type: "object",
-  properties: Object.assign({}, BaseItem.properties, {}),
+  properties: Object.assign({}, BaseItem.properties, {
+    changes: {
```

Comment:
Didn't use the word options, as it is a different schema to order options. Feel that is confusing.

Can't use `meta` as it is more than just meta, you need it to calculate the entire price.

Opted for `changes`.

Reply:
Yeah changes isn't bad, better than `additions` which was my first though. The only other thing I can think of is `subItems`, probably more representative, but kind of a crappy name.

Reply:
I don't mind `subItems` actually